### ISSUE TITLE
Build Squirrel.Windows issues

### ISSUE THREAD

**Issue by Alonzzzo2** [Squirrel/Squirrel.Windows#1633]
I downloaded the source code and I encounter some issues preventing me from successfully building the solution:
* update.exe is missing: When running this:
`cd "C:\Users\alon.dean\source\repos\Squirrel.Windows\build\Release\net45\"
      "C:\Users\alon.dean\.nuget\packages\ilrepack\1.26.0\tools\ILRepack.exe" /internalize /out:Update.exe.tmp Update.exe WpfAnimatedGif.dll SharpCompress.dll Microsoft.Web.XmlTransform.dll Splat.dll DeltaCompressionDotNet.dll DeltaCompressionDotNet.MsDelta.dll Squirrel.dll NuGet.Squirrel.dll Mono.Cecil.dll
      del "Update.exe"
      ren "Update.exe.tmp" "Update.exe"`
it fails, so I ran it in CMD and got 
"ERROR: Failed to load assembly Update.exe"

* Failed to load assembly Splat.dll

I'm getting some more errors but they may occur due to these 2 I mentioned.
Any assistance in building the solution would be great,
Thanks

**Comment by Alonzzzo2** [comment]
OK, I started compiling each and every project according to the solution's build order.
At first I had to install Windows SDK 8.1 from the VS installer (go to add remove programs, VS, and modify)

At 'setup', which is CPP, I found that ctype.h doesn't exist.
So [this](https://stackoverflow.com/questions/42258311/cannot-open-include-file-ctype-h-no-such-file-or-directory) helped me, especially the VS2017 comment.

Next, 'update' didn't build.
It said splat is missing. I noticed that it exists at the packages.config but it isn't installed according to the nuget package manager. So I installed it via the package manager, specifically version 1.6.2 as mentioned in the packages.config.

Now it successfully compiles.
I hope this will help other people in the future
